src/pages/index.js

```diff
@@ -10,31 +11,70 @@ import {
   canSeeOperativeWorkOrders,
 } from '@/utils/userPermissions'
 import { getQueryProps } from '@/utils/helpers/serverSideProps'
+import Search from '../components/Search/Search'
+import CurrentUserWrapper from '../components/WorkOrders/CurrentUserWrapper'
+import MobileWorkingWorkOrdersView from '../components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView'
+import WorkOrdersView from '@/components/WorkOrders/WorkOrdersView'
+import NewTabs from '../components/NewTabs/Index'
 
 const Home = ({ query }) => {
   const { user } = useContext(UserContext)
+  const router = useRouter()
 
   const [loading, setLoading] = useState(true)
 
+  const titles = ['Current Work Orders', 'Past Work Orders']
+
+  const handleTabClick = (index) => {
+    index === 1 && router.push('/oldjobs')
+  }
+
+  const ariaSelected = useMemo(() => {
+    return router.pathname === '/oldjobs' ? 1 : 0
+  }, [router.pathname])
+
   const HomeView = () => {
     if (user && canSeeWorkOrders(user)) {
       // Use saved filter preset in local storage as the default applied filters (if present)
       const defaultFilters = JSON.parse(
         localStorage.getItem('RH - default work order filters')
       )
-
       if (Object.entries(query).length === 0) {
         return (
-          <WorkOrdersView
-            pageNumber={1}
-            {...(defaultFilters && { query: defaultFilters })}
-          />
+          <>
+            <NewTabs
+              titles={titles}
+              onTabChange={handleTabClick}
+              ariaSelected={ariaSelected}
+            />
+            <WorkOrdersView
+              pageNumber={1}
+              {...(defaultFilters && { query: defaultFilters })}
+            />
+          </>
```

Comment:
Did you intent to add tabs here?

Reply:
Yes because I believe it won't show up on any desktop versions. I could be wrong though

Reply:
But why do you need it there?

Reply:
Harriet said it's an edge case but that we should try to ensure it's as user friendly as possible

Reply:
Remove tabs from desktop view